user-guide/Advanced_Modules/Dashboards_and_Low_Code_Apps/Feeds/Specifying_data_input_in_a_dashboard_URL.md

```diff
@@ -49,9 +56,9 @@ This JSON object has to have the following structure:
 
 - When you provide data in the (optional) *feedAndSelect* item, that data will be interpreted as if it was passed using the [legacy syntax](#legacy-syntax) described below.
 
-- When you provide data in the (optional) *feed* item, that data will only be used in the URL feed. It will not be used to select items in selection boxes on the dashboard.
+- When you provide data in the (optional) *feed* item, that data will only be used in the URL feed. It will not be used to select items in selection boxes on the dashboard or app page.
```

Comment:
The URL feed does not work in LCA yet. Although this is more an issue and we want this to work at one point.

Reply:
```suggestion
- When you provide data in the (optional) *feed* item, that data will only be used in the URL feed. It will not be used to select items in selection boxes on the dashboard.

  > [!NOTE]
  > The (optional) *feed* item is currently only available for passing data input in a dashboard URL.
```